src/compiler-debugging.md

```diff
@@ -185,6 +185,40 @@ stack backtrace:
 
 Cool, now I have a backtrace for the error!
 
+## Getting the the error creation location
+
+`-Z track-diagnostics` can help figure out where errors are emitted. It uses `#[track_caller]`
+for this and prints its location alongside the error:
+
+```bash
```

Comment:
```suggestion
```
```
I would not add this since what follows is not shell code, just console output... would indeed be nice to have colors matching rustc error messages